### DISCUSSION TITLE
Redirect `https://beta.ruff.rs/docs/rules/<rule code>` to `https://beta.ruff.rs/docs/rules/<rule name>`?

### DISCUSSION THREAD

**Discussion by harupy** [astral-sh/ruff#6804] [Ideas]
An idea that came to my mind while working on https://github.com/mlflow/mlflow/pull/9421. Is it possible to redirect `https://beta.ruff.rs/docs/rules/<rule code>` to `https://beta.ruff.rs/docs/rules/<rule name>`?

**Comment by zanieb** [comment]
Seems reasonable to me although after a brief look it seems a bit painful to do and it feels kind of low priority. Given that we're interested in re-categorizing the rules in the future, it may be worth tackling then instead so we don't have to maintain an even larger list of redirects.

**Comment by saippuakauppias** [comment]
> `https://beta.ruff.rs/docs/rules/<rule code>` 

It's not working now, I requested this here: https://github.com/astral-sh/ruff/issues/4710

**Comment by harupy** [comment]
I'm actually not sure if I still need this, closing.
